WordPress/src/main/res/values/strings.xml

```diff
@@ -1373,6 +1374,12 @@
 
     <string name="upload_finished_toast">Can\'t stop the upload because it\'s already finished</string>
 
+    <string name="link_dialog_title">Insert link</string>
+    <string name="link_dialog_button_remove_link">Remove Link</string>
+    <string name="link_dialog_edit_hint">http(s)://</string>
```

Comment:
Does this need to be added? I don't see it used anywhere in WPAndroid.

Reply:
As far as I understand, these are the resources used inside Aztec to generate the untranslated dialog in #10132. Aztec is imported as lib, not a subtree/submodule, so the code is not in the WordPress repository.
Though, I'm not familiar with Aztec, so it's totally possible that I'm completely wrong here :-) 
I think @mzorz and @daniloercoli are far more knowledgeable than me here :-)

Reply:
That's correct - for the time being it should suffice to duplicate the strings existing in Aztec on the WordPress android app. The strings will be overriden when the app is built so, given we pull translations for the WP app only (and not for Aztec), the translations will end up there and the dialog would be shown correctly translated. Thanks for this @loremattei @rachelmcr !